tests/test_pipelines_common.py

```diff
@@ -90,6 +90,22 @@ def test_save_load_local(self):
         max_diff = np.abs(output - output_loaded).max()
         self.assertLess(max_diff, 1e-5)
 
+    def test_signature(self):
```

Comment:
There's also `test_pipeline_call_implements_required_args` further down, feel free to merge them into one :)